moto/transcribe/models.py

```diff
@@ -0,0 +1,401 @@
+                "Type",
+            ],
+            "LIST": [
+                "MedicalTranscriptionJobName",
+                "CreationTime",
+                "StartTime",
+                "CompletionTime",
+                "LanguageCode",
+                "TranscriptionJobStatus",
+                "FailureReason",
+                "OutputLocationType",
+                "Specialty",
+                "Type",
+            ],
+        }
+        response_fields = response_field_dict[response_type]
+        response_object = self.gen_response_object()
+        if response_type != "LIST":
+            return {
+                "MedicalTranscriptionJob": {
+                    k: v
+                    for k, v in response_object.items()
+                    if k in response_fields and v is not None and v != [None]
+                }
+            }
+        else:
+            return {
+                k: v
+                for k, v in response_object.items()
+                if k in response_fields and v is not None and v != [None]
+            }
+
+    def advance_job_status(self):
+        # On each call advances the fake job status
+
+        if not self.transcription_job_status:
+            self.transcription_job_status = "QUEUED"
+        elif self.transcription_job_status == "QUEUED":
+            self.transcription_job_status = "IN_PROGRESS"
+            self.start_time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+            if not self.media_sample_rate_hertz:
+                self.media_sample_rate_hertz = 44100
+            if not self.media_format:
+                file_ext = self.media["MediaFileUri"].split(".")[-1].lower()
+                self.media_format = (
+                    file_ext if file_ext in ["mp3", "mp4", "wav", "flac"] else "mp3"
+                )
+        elif self.transcription_job_status == "IN_PROGRESS":
+            self.transcription_job_status = "COMPLETED"
+            self.completion_time = (datetime.now() + timedelta(seconds=10)).strftime(
+                "%Y-%m-%d %H:%M:%S"
+            )
+            self.transcript = {
+                "TranscriptFileUri": "https://s3.{}.amazonaws.com/{}/medical/{}.json".format(
+                    self._region_name,
+                    self._output_bucket_name,
+                    self.medical_transcription_job_name,
+                )
+            }
+
+
+class FakeMedicalVocabulary(BaseObject):
+    def __init__(
+        self, region_name, vocabulary_name, language_code, vocabulary_file_uri,
+    ):
+        self._region_name = region_name
+        self.vocabulary_name = vocabulary_name
+        self.language_code = language_code
+        self.vocabulary_file_uri = vocabulary_file_uri
+        self.vocabulary_state = None
+        self.last_modified_time = None
+        self.failure_reason = None
+        self.download_uri = "https://s3.us-east-1.amazonaws.com/aws-transcribe-dictionary-model-us-east-1-prod/{}/medical/{}/{}/input.txt".format(
```

Comment:
Is the region always the same for this URL, or is it dependent on `self._region_name`?

Reply:
Confirmed that that component of the URI truly varies by region.  Done.